packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-dropdown/gux-form-field-dropdown.tsx

```diff
@@ -95,6 +142,7 @@ export class GuxFormFieldDropdown {
       this.requiredObserver.disconnect();
     }
   }
+
   private renderScreenReaderText(
```

Comment:
I'm not sure if there's much point to having text within the legend into <gux-screenreader-beta> here. I think the condition is useful but I'm not sure otherwise what's the benefit. Anyone know?